DAQ_Simulation.py

```diff
@@ -10,21 +10,50 @@
 port = 5000
 
 
+possible_data = [
+    # {
+    #     'id': '421',
+    #     'data': '03FFBA00776F'
+    # },
+    # {
+    #     'id': '422',
+    #     'data': '68090000'
+    # },
+    # {
+    #     'id': '421',
+    #     'data': '01FFC000777F'
+    # },
+    # {
+    #     'id': '421',
+    #     'data': '00FFF70077FF'
+    # },
+    # {
+    #     'id': '601',
+    #     'data': '00010000'
+    # },
+    # {
+    #     'id': '6B0',
+    #     'data': '00010B90C710416C'
+    # },
+    {
+        'id': '501',
+        'data': '0F900EBF02A0029F'
+    },
+]
+
+
 async def send_data():
     while True:
-        await asyncio.sleep(1)
-        data = {
-                'accel': random.random() * 10,
-                'gyro': random.random() * 360,
-                'angle': random.random() * 180
-                }
+        await asyncio.sleep(0.01)
+        index = int(random.random() * len(possible_data))
+        data = possible_data[index]
         await server.send_data(data)
 
 
 async def run_server(ip, port):
+    print('Hello?')
     wait_task = asyncio.ensure_future(send_data())
     server_task = asyncio.ensure_future(server.get_server('127.0.0.1', 5000))
-    print(f'Serving websocket server on {ip}:{port} ...')
```

Comment:
Is the `ip` and `port` documented somewhere else so people new to the project know where to find this information?

Reply:
Yes, it is located on the wiki [here](http://purdueelectricracing.com/wiki/index.php/Wireless_Telemetry)